dev/src/document.ts

```diff
@@ -52,14 +49,20 @@ export class DocumentSnapshotBuilder {
   /** The time when this document was last updated. */
   updateTime?: Timestamp;
 
+  /**
```

Comment:
Nit: Since this doesn't describe what "constructor" is doing, this comment should use `/* */` or `//`. As it stands currently, the comment becomes part of the API documentation.